reglement-interieur.md

```diff
@@ -146,7 +146,9 @@ vote à la majorité des deux tiers de ses membres.
 
 Toute personne souhaitant adhérer à « Zeste de Savoir » doit s’acquitter d'une
 cotisation. Le montant de la cotisation est fixé lors de l'Assemblée Générale
-Ordinaire sur proposition du Conseil d'Administration. Le montant de la
+Ordinaire sur proposition du Conseil d'Administration. Le montant de la cotisation 
+peut être fixé exceptionnellement par le conseil d'administration seul sur demande 
+explicite de l'assemblée générale. Le montant de la
```

Comment:
Dans le PV de l'AG il y a ça de noté: 

"En conséquence de l’absence du Président et du Trésorier [le rapport d’activité] sera fait après l’AGO par le Conseil d’Administration (CA) une fois que le nouveau Bureau aura récupéré tous les documents et accès aux comptes de l’association. De même pour la fixation du montant de la cotisation pour l’année 2016."

donc je pense que oui